### PR TITLE
Alpine linux instructions

### PR DESCRIPTION
**Pull request by osfameron**
Supersedes #88

### CODE REVIEW

File: modules/hello-world/pages/start-using-sdk.adoc [hunk 1]
```diff
@@ -16,19 +16,64 @@ The Couchbase PHP SDK is compatible with PHP 7.3+.
 
 == Installing the SDK
 
-Before installing the PHP SDK, you must install the xref:3.2@c-sdk:hello-world:start-using-sdk.adoc[C SDK -- _libcouchbase_ (LCB)];
+The Couchbase PHP SDK depends on the xref:3.2@c-sdk:hello-world:start-using-sdk.adoc[C SDK -- _libcouchbase_ (LCB)];
 version 3.2.0 or higher is required for PHP Client 3.2.
 
-After installing the C SDK, install the PHP SDK through your PHP distribution's pecl command:
+On **Linux** and **macOS**, `libcouchbase` is installed separately as a system library.
```
Comment (maria-robobug) [inline modules/hello-world/pages/start-using-sdk.adoc]: Should this be `MacOS`?
```suggestion
On **Linux** and **MacOS**, `libcouchbase` is installed separately as a system library.
```

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 2]
```diff
@@ -109,23 +139,24 @@ $ composer require couchbase/couchbase
 
 === Note on Extension Dependencies
 
-The Couchbase SDK depends on the JSON module, which must be loaded before the SDK.
+The Couchbase SDK depends on the https://www.php.net/manual/en/json.installation.php[JSON extension],
+which must be loaded before the SDK. However, it is already included on PHP 8.0.0+ as a core extension.
```
Comment (maria-robobug) [inline modules/project-docs/pages/sdk-release-notes.adoc:141]:
```suggestion
The Couchbase SDK depends on the https://www.php.net/manual/en/json.installation.php[JSON extension], which must be loaded before the SDK. 
However, it is already included on PHP 8.0.0+ as a core extension.
```

**Review by maria-robobug — APPROVED** [review]
👍  Nice one! Just minor comment/suggestion.

File: modules/project-docs/pages/sdk-release-notes.adoc (hunk 2 above)
Comment (RichardSmedley) [reply modules/project-docs/pages/sdk-release-notes.adoc:141]: :+1:

**Review by RichardSmedley — APPROVED** [review]
LGTM   :+1:

**Pull request merged**